<ui version="4.0" >
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>transferOrders</class>
 <widget class="QWidget" name="transferOrders" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>List Open Transfer Orders</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_4" >
     <property name="margin" >
      <number>12</number>
     </property>
     <property name="spacing" >
      <number>12</number>
     </property>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <layout class="QGridLayout" name="gridLayout_3" >
         <item row="0" column="0" >
          <widget class="WarehouseGroup" name="_srcWarehouse" >
           <property name="title" >
            <string>Source Site:</string>
           </property>
           <property name="fixedSize" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="WarehouseGroup" name="_destWarehouse" >
           <property name="title" >
            <string>Destination Site:</string>
           </property>
           <property name="fixedSize" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Status</string>
           </property>
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" colspan="2" >
             <widget class="QRadioButton" name="_allStatuses" >
              <property name="text" >
               <string>All Statuses</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QRadioButton" name="_selectedStatus" >
              <property name="text" >
               <string>Selected:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <layout class="QHBoxLayout" name="horizontalLayout" >
              <item>
               <widget class="XComboBox" name="_statuses" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text" >
                  <string>Unreleased</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Open</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>13</width>
           <height>69</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="_close" >
             <property name="text" >
              <string>&amp;Close</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_print" >
             <property name="text" >
              <string>&amp;Print</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QFrame" name="_frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="_transferOrdersLit" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text" >
             <string>Transfer Orders:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="XTreeWidget" name="_to" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>5</number>
              </property>
              <item>
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="_new" >
                  <property name="enabled" >
                   <bool>true</bool>
                  </property>
                  <property name="text" >
                   <string>&amp;New</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_edit" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>&amp;Edit</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_view" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>&amp;View</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_delete" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>&amp;Delete</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="_release" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>Release</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_issue" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>Issue Stock</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="_copy" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string>Co&amp;py</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>WarehouseGroup</class>
   <extends>QGroupBox</extends>
   <header>warehousegroup.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_srcWarehouse</tabstop>
  <tabstop>_destWarehouse</tabstop>
  <tabstop>_to</tabstop>
  <tabstop>_new</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_view</tabstop>
  <tabstop>_delete</tabstop>
  <tabstop>_issue</tabstop>
  <tabstop>_copy</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_to</sender>
   <signal>valid(bool)</signal>
   <receiver>_view</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>43</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>720</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>transferOrders</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>775</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>289</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_selectedStatus</sender>
   <signal>toggled(bool)</signal>
   <receiver>_statuses</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>526</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>632</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
